backend/ibutsu_server/db/models.py

```diff
@@ -164,7 +177,66 @@ class WidgetConfig(Model, ModelMixin):
     navigable = Column(Boolean, index=True)
     params = Column(mutable_json_type(dbtype=PortableJSON()))
     project_id = Column(PortableUUID(), ForeignKey("projects.id"), index=True)
+    dashboard_id = Column(PortableUUID(), ForeignKey("dashboards.id"), index=True)
     title = Column(Text, index=True)
     type = Column(Text, index=True)
     weight = Column(Integer, index=True)
     widget = Column(Text, index=True)
+
+
+class User(Model, ModelMixin):
+    __tablename__ = "users"
+    email = Column(Text, index=True, nullable=False, unique=True)
+    password = Column(Text, nullable=False)
```

Comment:
From a cursory google search, thought on using `bcrypt` for storing passwords? 
https://uniwebsidad.com/libros/explore-flask/chapter-12/storing-passwords

Reply:
Yup, we  can. I think I'll leave the actual implementation until we implement auth though.

Reply:
Yup, that sounds good to me.

Reply:
I did it anyway :-)